### DISCUSSION TITLE
ERROR @wdio/utils:initialiseServices: Error: Couldn't find plugin "intercept" service, neither as wdio scoped package "@wdio/intercept-service" nor as community package "wdio-intercept-service"

### DISCUSSION THREAD

**Discussion by AmritaDatta** [webdriverio/webdriverio#8683] [General]
Hello Experts,
I couldn't post in the GITTER, that's why I'm writing here. Hope you won't mind.
I'm getting the below error when I try to run my WDIO integration test locally. Any help would be much appreciate and please let me know if you need any further detail from my side. Thank you.
`ERROR @wdio/utils:initialiseServices: Error: Couldn't find plugin "intercept" service, neither as wdio scoped package "@wdio/intercept-service" nor as community package "wdio-intercept-service".`

package.json:
"devDependencies": {
        "@ui5/cli": "^2.14.3",
        "dateformat": "^3.0.3",
        "env-cmd": "^10.1.0",
        "eslint": "^7.32.0",
        "karma": "6.3.17",
        "karma-sinon": "^1.0.5",
        "karma-chrome-launcher": "^3.1.1",
        "karma-coverage": "^2.2.0",
        "karma-junit-reporter": "^2.0.1",
        "karma-ui5": "2.4.0",
        "karma-qunit": "^4.1.2",
        "qunitjs": "^2.4.1",
        "rimraf": "^3.0.2",
        "start-server-and-test": "^1.14.0",
        "nyc": "^15.1.0",
        "ui5-middleware-livereload": "^0.4.1",
        "ui5-middleware-servestatic": "^0.3.2",
        "ui5-middleware-simpleproxy": "^0.7.5"
    },

**Comment by christian-bromann** [comment]
@AmritaDatta can you please elaborate what your question is? The error says that you don't have the package installed which you indeed don't have given your snippet of the package.json.

**Comment by AmritaDatta** [comment]
Hi @christian-bromann 
Thank you for your reply. We're running wdio integration tests to check our Pull request and master pipeline in Jenkins. Currently all the PRs are red in the pipeline. I know it's bit complecated to explain. 
And I tried to run it in locally, got the the error that I mentiond. Not sure where the problem is. Thanks

**Comment by AmritaDatta** [comment]
Yes... As far as I can remember, i removed packege-lock. json file and re
installed node and npm, upgraded chrome version as well.
